ts/features/wallet/bancomatpay/screen/BPayDetailScreen.tsx

```diff
@@ -0,0 +1,120 @@
+import { Dispatch } from "redux";
+import { Label } from "../../../../components/core/typography/Label";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import DarkLayout from "../../../../components/screens/DarkLayout";
+import I18n from "../../../../i18n";
+import { deleteWalletRequest } from "../../../../store/actions/wallet/wallets";
+import { GlobalState } from "../../../../store/reducers/types";
+import { BPayPaymentMethod } from "../../../../types/pagopa";
+import { showToast } from "../../../../utils/showToast";
+import PaymentMethodCapabilities from "../../component/PaymentMethodCapabilities";
+import { useRemovePaymentMethodBottomSheet } from "../../component/RemovePaymentMethod";
+import bPayImage from "../../../../../img/wallet/cards-icons/bPay.png";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import BPayCard from "../component/BPayCard";
+
+type NavigationParams = Readonly<{
+  bPay: BPayPaymentMethod;
+}>;
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps> &
+  NavigationInjectedProps<NavigationParams>;
+
+const styles = StyleSheet.create({
+  cardContainer: {
+    height: 235,
+    width: "100%",
+    position: "absolute",
+    top: 16,
+    zIndex: 7,
+    elevation: 7,
+    alignItems: "center"
+  },
+  headerSpacer: {
+    height: 172
+  },
+  cancelButton: {
+    borderColor: IOColors.red,
+    width: "100%"
+  }
+});
+
+const UnsubscribeButton = (props: { onPress?: () => void }) => (
+  <Button bordered={true} style={styles.cancelButton} onPress={props.onPress}>
+    <Label color={"red"}>{I18n.t("wallet.bancomat.details.removeCta")}</Label>
+  </Button>
+);
+
+/**
+ * Detail screen for a Bancomat Pay
+ * @constructor
+ */
+const BPayDetailScreen: React.FunctionComponent<Props> = props => {
+  const bPay: BPayPaymentMethod = props.navigation.getParam("bPay");
+
+  const { present } = useRemovePaymentMethodBottomSheet({
+    icon: bPayImage,
+    caption: I18n.t("wallet.methods.bancomatPay.name")
+  });
+  return (
+    <DarkLayout
+      bounces={false}
+      contextualHelp={emptyContextualHelp}
+      title={I18n.t("wallet.methods.bancomatPay.name")}
+      faqCategories={["wallet_methods"]}
+      allowGoBack={true}
+      topContent={<View style={styles.headerSpacer} />}
+      gradientHeader={true}
+      hideHeader={true}
+    >
+      <View style={styles.cardContainer}>
+        <BPayCard
+          phone={bPay.caption}
+          bankName={
+            bPay.abiInfo?.name ??
+            bPay.info.bankName ??
+            I18n.t("wallet.methods.bancomatPay.name")
+          }
+          abiLogo={bPay.abiInfo?.logoUrl}
+        />
+      </View>
+      <View spacer={true} extralarge={true} />
+      <View style={IOStyles.horizontalContentPadding}>
+        <PaymentMethodCapabilities paymentMethod={bPay} />
```

Comment:
Based on https://www.pivotaltracker.com/story/show/175883472/comments/220975851, could you change the `BPay` behaviour in `isPaymentMethodSupported` in order to return `arriving`?

Reply:
Yes, no problem!